.github/workflows/main.yml

```diff
@@ -320,7 +322,9 @@ jobs:
             cd /workspace
             source container-env.sh
             python3 .github/workflows/build_and_test.py ${{ runner.os }} x64 ${{ matrix.compiler }} ${{ matrix.version }}
-        run: docker exec build-container bash -c "$CMD"
+        run:
+          python3 -m pip install packaging==25.0
```

Comment:
This needs to go into the list of commands in `$CMD`, otherwise it will be executed on the host, not in the scope of the container. Also, this needs to be done only once, best in the `Install (Linux, containerized)` step.